scripts/protocgen-any.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
How about putting this script into `contrib`?

Reply:
Well `protocgen.sh` is in `scripts`. What is the rationale for `contrib`? For me `scripts` seems more self-explanatory. `contrib` suggests 3rd party contributions not build scripts

Reply:
Well, that makes sense :+1: